pkg/operation/botanist/component/extensions/dnsrecord/dnsrecord.go

```diff
@@ -146,8 +146,8 @@ func (c *dnsRecord) deploy(ctx context.Context, operation string) (extensionsv1a
 	mutateFn := func() error {
 		if c.values.AnnotateOperation {
 			metav1.SetMetaDataAnnotation(&c.dnsRecord.ObjectMeta, v1beta1constants.GardenerOperation, operation)
-			metav1.SetMetaDataAnnotation(&c.dnsRecord.ObjectMeta, v1beta1constants.GardenerTimestamp, TimeNow().UTC().String())
 		}
+		metav1.SetMetaDataAnnotation(&c.dnsRecord.ObjectMeta, v1beta1constants.GardenerTimestamp, TimeNow().UTC().String())
```

Comment:
For DNSRecord, this is taken care already here, right? 
https://github.com/gardener/gardener/blob/dcf0ea638de5b5320c0d88370e4b74ff8e8052ff/pkg/operation/botanist/component/extensions/dnsrecord/dnsrecord.go#L192

Reply:
When the last operation is not `Succeeded` then [this](https://github.com/gardener/gardener/blob/a02928054df88b8cf5e60104ace5ca32f1d0ab21/pkg/operation/botanist/component/extensions/dnsrecord/dnsrecord.go#L185) statement will be executed and if doesnot have `gardener.cloud/operation` set then timestamp will never be updated.